\begin{titlepage}
\vspace*{10mm}
	\begin{center}
		\rule[0.5ex]{\linewidth}{2pt}\vspace*{-\baselineskip}\vspace*{3.2pt}
		\rule[0.5ex]{\linewidth}{1pt}\\[\baselineskip]

{\LARGE{\textbf{An integrated metagenomic approach to investigating disease heterogeneity in sepsis due to community acquired pneumonia}}}\\[4mm]

\rule[0.5ex]{\linewidth}{1pt}\vspace*{-\baselineskip}\vspace{3.2pt}
\rule[0.5ex]{\linewidth}{2pt}\\
\vspace{20mm}

{\Large Cyndi Goh}\\

\vspace{6mm}
\includegraphics[scale=0.3]{./oxford_logo-eps-converted-to.pdf}\\
\vspace{6mm}

{\Large Linacre College}\\
\vspace{4mm}
{\large\textsc{University of Oxford}}\\
\vspace{10mm}

\begin{minipage}{10cm}
\begin{center}
{\large A thesis submitted in partial fulfilment of the requirements for the degree of Doctor of Philosophy}
\end{center}
\end{minipage}\\

\vspace{9mm}
{\large\textsc{Michaelmas Term, 2019}}
\vspace{12mm}

\end{center}
\end{titlepage}

\newpage

\chapter*{Abstract} %Creates chapter title but doesn't number
\addcontentsline{toc}{chapter}{\numberline{}Abstract} %Add abstract to table of contents
\thispagestyle{plain}
\pagenumbering{roman} \setcounter{page}{1}
\onehalfspacing
\vspace*{0.4cm}
\begin{center}
{\large\bf An integrated metagenomic approach to investigating disease heterogeneity in sepsis due to community acquired pneumonia} \\
{Cyndi Goh, Linacre College, Michaelmas Term, 2019}\\
\vspace*{0.4cm}
{\itshape A thesis submitted in partial fulfilment of the requirements
for the degree of Doctor of Philosophy of the University of Oxford} \\
\vspace*{0.4cm}
\end{center}

\singlespacing

Your abstract text goes here.

Your abstract text goes here.

Your abstract text goes here.

\newpage
\chapter*{Acknowledgements}
\addcontentsline{toc}{chapter}{\numberline{}Acknowledgements}
\thispagestyle{plain}
\noindent
\singlespacing

Firstly, I would like to thank my two supervisors Julian Knight and Ellie Barnes. Julian has made the Knight group a very happy place to spend four years of my life and I am grateful to him for his patience and guidance, and always having time for me despite the busyness of running a large group. I have particularly appreciated Ellie's wisdom and support during a time of illness.

Members of the Knight group have also provided me with immense support. Special thanks must go to Katie Burnham who has been endlessly willing to share scripts, troubleshoot problems big and small and point me in the right direction. 

I have also been enormously grateful for the care and support I received from Dr Rose Freeman, Dr Evie Kemp, Dr Desi Choi, and Dr Rebecca Knowles Bevis. This DPhil would not have been possible without you. 

Finally, thanks must go to my cycling buddies Becca Kearney and Mimi Harrison who have helped me maintain my sanity on two-wheeled trips involving cake. My family, Mum, Wendi and Jeremy have been an endless source of love and support. And of course, Oscar who never fails to cheer me up with licks and cuddles.







\newpage
\chapter*{Declarations}
\addcontentsline{toc}{chapter}{\numberline{}Declarations}
\thispagestyle{plain}
\noindent

I declare that, unless otherwise stated, all work presented in this thesis is my own. Several aspects of the study relied upon collaboration where part of the work was conducted with or by others.

\textbf{Study recruitment}: The Genomic Advances in Sepsis (GAinS) study began recruiting patients in 2005 from 34 intensive care units across the UK. From October 2015, I have been responsible for liaising with research nurses, providing supplies, maintaining sample records, and collecting and processing samples.

\textbf{Pathogen probes}: Probe design was carried out by Azim Ansari. Curation of meningitis pathogen sequences from RefSeq was performed by Ivo Elliott. Curation of additional pneumonia pathogen sequences was performed by myself. 

\textbf{Metagenomics}: I carried out the nucleic acid extractions (supervised by Anthony Brown) and library preparation (supervised by Amy Trebes) for the initial two optimisation experiments and 32 metagenomic samples. Nucleic acid extractions for the subsequent samples were carried out by Anthony Brown and George Macintyre. Library preparation for the remainining samples were carried out by Hubert Slawinski and Mariateresa de Cesare. Data processing and analysis was performed in collaboration with Tanya Golubchik.

\textbf{Cardiac surgery patients}: The cardiac surgery patients were recruited by Eduardo Svoren and Bart's and the London NHS Trust.

\textbf{Axiom microbiome array}: Library preparation was performed by Hannah Matten.

\textbf{Gene expression}: RNA sample processing was performed by Yuxin Mi, Alice Allcock, Katie Burnham, Emma Davenport, Jayachandran Radhakrishnan, Narelle Magueri, and Ashley Thorpe. Microarray gene expression data was generated by the Wellcome Trust Sanger Institute (WTSI) and Wellcome Centre for Human Genetics (WCHG) Core Genomics facilities.

\textbf{HLA work}: probe design was carried out by Azim Ansari. Library preparation for HLA enrichment experiments were performed by Mariateresa de Cesare. DNA samples were processed by myself, Yuxin Mi, Andrew Kwok, Alice Allcock, Katie Burnham, Jayachandran Radhakrishnan, Emma Davenport, Anna Rautanen, and Tara Mills. Genotyping data was generated by the WCHG and WTSI Core Genomics facilities. Genotyping QC was performed by Emma Davenport and Katie Burnham. SNP2HLA imputation was performed by Justin Whalley.

\newpage

\chapter*{Submitted Abstracts}
\addcontentsline{toc}{chapter}{\numberline{}Submitted Abstracts}
\thispagestyle{plain} %Removes the header

\noindent
\textbf{Quest for the one true test: enrichment-based metagenomic sequencing in paediatric meningitis and adult sepsis}\\
Oral: Applied Bioinformatics and Public Health Microbiology (2019)\\
T Golubchik, \textbf{C Goh}, MA Ansari, M de Cesare, A Trebes, D Bonsall, A Brown, M Sadarangani, P Piazza, K Jolley, I Elliott, C Ip, H Slawinski, A Coxon, G Meddaugh, P Hutton, CJ Hinds, E Barnes, AJ Pollard, JC Knight, R Bowden\\

\textbf{Using \textit{in vivo} eQTL interactions to identify the regulatory drivers of variation in the transcriptomic response to sepsis}\\
Poster: Immunogenomics of Disease: Accelerating to Patient Benefit (2019)\\
EE Davenport, KL Burnham, \textbf{C Goh}, J Radhakrishnan, P Hutton, TC Mills, A Rautanen, AC Gordon, N Soranzo, AVS Hill, CJ Hinds, S Raychaudhuri, JC Knight\\

\textbf{Using \textit{in vivo} eQTL interactions to identify the genetic drivers of the transcriptomic response to sepsis}\\
Poster: American Society of Human Genetics (2018)\\
EE Davenport, KL Burnham, \textbf{C Goh}, J Radhakrishnan, P Hutton, TC Mills, A Rautanen, AC Gordon, N Soranzo, AVS Hill, CJ Hinds, S Raychaudhuri, JC Knight\\

\newpage
\chapter*{Associated Publications}
\addcontentsline{toc}{chapter}{\numberline{}Associated Publications}
\thispagestyle{plain}

\noindent
\textbf{Enhanced understanding of the host-pathogen interaction in sepsis: new opportunities for omic approaches}\\
The Lancet Respiratory Medicine (2017)\\
\textbf{C Goh}, JC Knight\\

\textbf{Targeted metagenomic sequencing enhances the identification of pathogens associated with acute infection}\\
Nature Microbiology (under review), bioRxiv (2019)\\
\textbf{C Goh}, T Golubchik, MA Ansari, M de Cesare, A Trebes, I Elliott, D Bonsall, P Piazza, A Brown, H Slawinski, N Martin, S Defres, MJ Griffiths, JE Bray, MC Maiden, P Hutton, CJ Hinds, T Solomon, E Barnes, AJ Pollard, M Sadarangani, JC Knight, R Bowden\\

\textbf{Epstein-Barr virus reactivation in sepsis is associated with an immunosuppressed host transcriptomic endotype}\\
Critical Care Medicine (under review)\\
\textbf{C Goh}, KL Burnham, MA Ansari, M de Cesare, T Golubchik, P Hutton, LE Overend, EE Davenport, CJ Hinds, R Bowden, JC Knight\\


\newpage
\addcontentsline{toc}{chapter}{\numberline{}Contents}
\tableofcontents

\newpage
\listoffigures
\addcontentsline{toc}{chapter}{\numberline{}List of Figures}

\newpage
\listoftables
\addcontentsline{toc}{chapter}{\numberline{}List of Tables}

\chapter*{Abbreviations}
\addcontentsline{toc}{chapter}{\numberline{}Abbreviations}

\begin{longtable}[l]{l l}
		\textbf{APACHE-II} & Acute physiology and chronic health evaluation II\\
		\textbf{AST} & Antimicrobial stewardship team\\
		\textbf{AUC} & Area under the curve\\
		\textbf{bp} & Base pairs\\
		\textbf{BWA} & Burrows-Wheeler Aligner\\
		\textbf{CAP} & Community-acquired pneumonia\\
		\textbf{cDNA} & Complementary deoxyribonucleic acid\\
		\textbf{CF} & Combined with fragmentation\\
		\textbf{ChiMES} & Childhood meningitis and encephalitis study\\
		\textbf{CI} & Confidence interval\\
		\textbf{CLiMax} & Confidence likelihood maximisation\\
		\textbf{CMV} & Cytomegalovirus\\
		\textbf{CnoF} & Combined with no fragmentation\\
		\textbf{CSF} & Cerebrospinal fluid\\
		\textbf{ddPCR} & Digital droplet PCR\\
		\textbf{DNA} & Deoxyribonucleic acid\\
		\textbf{DNase} & Deoxyribonuclease\\
		\textbf{dsDNA} & Double-stranded deoxyribonucleic acid\\
		\textbf{dsRNA} & Double-stranded ribonucleic acid\\
		\textbf{EBV} & Epstein-Barr virus\\
		\textbf{eCRF} & Electronic case record form\\
		\textbf{EDTA} & Ethylenediaminetetraacetic acid\\
		\textbf{ELISA} & Enzyme-linked immunosorbent assay\\
		\textbf{EPIC} & Aetiology of pneumonia in the community\\
		\textbf{eQTL} & Expression quantitative trait loci\\
		\textbf{ERCC} & External ribonucleic acid controls consortium\\
		\textbf{ESI-MS} & Electrospray ionisation mass spectrometry\\
		\textbf{FDR} & False discovery rate\\
		\textit{\textbf{FER}} & Fes/Fps related tyrosine kinase\\
		\textbf{GAinS} & Genomic advances in sepsis\\
		\textbf{GWAS} & Genome-wide association study\\
		\textbf{GWLS} & Genome-wide linkage study\\
		\textbf{HBV} & Hepatitis B virus\\
		\textbf{HCV} & Hepatitis C virus\\
		\textbf{HHV6} & Human herpesvirus 6\\
		\textbf{HIV} & Human immunodeficiency virus\\
		\textbf{HLA} & Human leukocyte antigen\\
		\textbf{HR} & Hazard ratio\\
		\textbf{HSV} & Herpes simplexvirus\\
		\textbf{HWE} & Hardy-Weinberg equilibrium\\
		\textbf{ICU} & Intensive care unit\\
		\textbf{IPA} & Ingenuity pathway analysis\\
		\textbf{IRF} & Interferon regulatory factor\\
		\textbf{MAF} & Minor allele frequency\\
		\textbf{MAFFT} & Multiple alignment using fast Fourier transform\\
		\textbf{MALDI-TOF} & Matrix-associated laser desorption/ionisation-time of flight\\
		\textbf{MARS} & Molecular diagnosis and risk stratification of sepsis\\
		\textbf{MHC} & Major histocompatibility complex\\
		\textbf{MiDAS} & Microbial detection analysis software\\
		\textbf{miRNA} & Micro ribonucleic acid\\
		\textbf{MOSAIC} & Mechanisms of severe acute influenza consortium\\
		\textbf{MR} & Misclassification rate\\
		\textbf{NCBI} & National centre for biotechnology information\\
		\textbf{NGS} & Next-generation sequencing\\
		\textbf{NIBSC} & National institute for biological standards and control\\
		\textbf{NS1} & Non-structural protein 1\\
		\textbf{nt} & Nucleotide\\
		\textbf{PBWT} & Positional Burrows-Wheeler Transform\\
		\textbf{PC} & Principal component\\
		\textbf{PCA} & Principal component analysis\\
		\textbf{PCR} & Polymerase chain reaction\\
		\textbf{pHBV} & Hepatitis B virus plasmid\\
		\textbf{PRR} & Pattern recognition receptor\\
		\textbf{REALISM} & Reanimation low immune status markers\\
		\textbf{RF} & Random forest\\
		\textbf{rMLST} & Ribosomal multilocus sequence typing\\
		\textbf{RNA} & Ribonucleic acid\\
		\textbf{ROC} & Receiver operating curve\\
		\textbf{rps} & Ribosomal protein subunit\\
		\textbf{rRNA} & Ribosomal ribonucleic acid\\
		\textbf{RSV} & Respiratory syncytial virus\\
		\textbf{SIRS} & Systemic inflammatory response syndrome\\
		\textbf{SNP} & Single nucleotide polymorphism\\
		\textbf{SOFA} & Sequential organ failure assessment\\
		\textbf{SRS} & Sepsis response signature\\
		\textbf{ssRNA} & Single stranded RNA\\
		\textbf{STOP-HCV} & Stratified medicine to optimise the treatment of patients with hepatitis C virus infection\\
		\textbf{SURPI} & Sequence-based ultra-rapid pathogen identification\\
		\textbf{T1DGC} & Type 1 diabetes genetics consortium\\
		\textbf{TTV} & Torque teno virus\\
		\textbf{UK} & United Kingdom\\
		\textbf{USA} & United States of America\\
		\textbf{VANISH} & Vasopressin vs norepinephrine as initial therapy in septic shock\\
		\textbf{VCA} & Viral capsid antigen\\
		\textbf{VMR} & Viral multiplex reference\\
		\textit{\textbf{VPS13A}} & Vacuolar protein sorting 13 homolog A\\
		\textbf{vsn} & Variance stabilisation and normalisation\\
		\textbf{VZV} & Varicella zoster virus\\
		\textbf{WCHG} & Wellcome Centre for Human Genetics\\
		\textbf{WTSI} & Wellcome Trust Sanger Institute\\
		
		
		

\end{longtable}