\chapter{Introduction}
\label{ch:Introduction}
\textit{This chapter presents the aims of this thesis, and provides an overview of pre-existing knowledge relevant to these goals}

\startcontents[chapters]{\vspace{-1.4cm}}
\singlespacing
\printcontents[chapters]{}{1}{\section*{ }\setcounter{tocdepth}{1}}
\doublespacing
\vspace{0.5cm}

The overall objective of this thesis is to understand the role of microbiological aetiology in contributing to the observed heterogeneous host response in sepsis due to community acquired pneumonia (CAP). Sepsis is defined as life-threatening organ dysfunction caused by a dysregulated host response to infection \parencite{Singer2016}. It is a clinically heterogeneous syndrome with the pathogen as one source of heterogeneity. Clinical microbiology fails to provide a diagnosis in 60\% of cases, suggesting that alternative methods such as clinical metagenomics are required for improved diagnostics. This work is conducted as part of the UK Genomic Advances in Sepsis study (GAinS, http://ukccggains.com). 

In this thesis, I will describe the development of a library preparation method suitable for the metagenomic sequencing of both RNA- and DNA-based pathogens from plasma. I will integrate metagenomic, PCR-based and clinical microbiology data to improve microbiological phenotyping in the GAinS CAP sepsis cohort. Finally, I will apply this improved microbiological phenotyping to transcriptomic and genomic datasets to better characterise disease heterogeneity. 

In this chapter, I will summarise the existing literature as it relates to the work to be described in this thesis. First, I will discuss the sources of sepsis heterogeneity and how this is limiting progression in sepsis research. Next, I will describe how better understanding of the host-pathogen interaction can substantially enhance, and in turn benefit from, current and future application of omics-based approaches to understand the host response in sepsis. I will discuss how clinical metagenomics provides new opportunities in clinical microbiology and describe the applications of clinical metagenomics. Finally, I will detail the specific aims of this thesis. 

\section{Sepsis heterogeneity}
The increasing global burden of sepsis is associated with an unacceptably high mortality rate \parencite{Vincent2014}, long-term disability \parencite{Barnato2011} and shortened life expectancy \parencite{Cuthbertson2013}. Although the importance of early recognition and prompt treatment cannot be overemphasised \parencite{Seymour2017}, there remains a significant cohort of sepsis patients who progress to develop a severely dysregulated host response, with failure of vital organs, for whom supportive care in an intensive care unit (ICU) cannot prevent death. For such patients, it has been postulated that some form of adjunctive therapy designed, for example, to modulate the immune response, might improve outcomes. Disappointingly however, multiple clinical trials of such therapies have been uniformly negative \parencite{Marshall2014}. Contributory factors include the limitations of animal models \parencite{Buras2005}, suboptimal clinical trials, the poorly understood pathophysiology of this ill-defined clinical syndrome and, perhaps most importantly, the substantial variation in the sepsis response both within and between individuals. 
\subsection{Factors influencing heterogeneity}
Heterogeneity in sepsis is driven by a combination of factors (Figure \ref{fig:heterogeneity} A) and may manifest as individual differences in the clinical and molecular response to infection (Figure \ref{fig:heterogeneity} B). 

\FloatBarrier
\begin{figure}[htbp]
\centering
\includegraphics[scale=0.8]{./Introduction/Images/heterogeneity.pdf}
\caption[Heterogeneity in sepsis]{\textbf{Heterogeneity in sepsis.} Heterogeneity in the host response to sepsis can be resolved into clinically relevant endotypes. \textbf{A.} An interaction between host, environmental, and microbial factors leads to heterogeneity in the individual host response. \textbf{B.} This heterogeneity may be observed in the molecular and/or physiological response to sepsis, and may be resolved into patient subgroups or endotypes with clinically relevant and distinct disease phenotypes.}
\label{fig:heterogeneity}
\end{figure}

\subsection{Failure of clinical trials}
There has been overwhelming failure of over 100 Phase II and Phase III randomised clinical trials of therapies targeting the dysregulated inflammatory response in sepsis \parencite{Marshall2014}. These include drugs which:
\begin{enumerate}
\item Non-selectively suppress inflammation (e.g. ibuprofen \parencite{Bernard1997})
\item Neutralize microbial products (e.g. human anti-endotoxin monoclonal antibody \parencite{McCloskey1994})
\item Neutralize host inflammatory mediators (e.g. tumour necrosis factor receptor antagonist \parencite{Cohen1996})
\item Non-selectively target inflammatory mediators (e.g. intravenous immunoglobulins \parencite{Werdan2007})
\item Stimulate immune function (e.g. granulocyte colony-stimulating factor \parencite{Marti-Carvajal2012})
\item Have anticoagulant function (e.g. activated protein C \parencite{Root2003})
\end{enumerate}

Given the substantial heterogeneity seen in sepsis at both molecular and physiological levels, it is unsurprising that these trials have failed. All have used simple physiological parameters to identify the at-risk population and studied a reduction in mortality rate as a primary outcome. 

\subsection{Precision medicine approaches}
"Precision medicine" has been defined as \textit{"treatments targeted to the needs of individual patients on the basis of genetic, biomarker, phenotypic or psychosocial charaacteristics that distinguish a given patient from other patients with similar clinical presentations"} \parencite{Jameson2015}. 

In the context of sepsis, several examples indicate how a precision medicine approach may hold potential for benefit in clinical trials of sepsis therapy. In the example of drotrecogin alfa (recombinant human activated protein C), underlying host genetics may influence treatment outcome. A genome-wide association study (GWAS) of treatment response in sepsis patients receiving drotrecogin alfa identified several single nucleotide polymorphism (SNP) combinations in biologically relevant genes which were associated with a positive treatment outcome \parencite{Man2013}. In the top combination of three SNPs (observed in 26\% of the cohort), an large absolute risk reduction in 28-day mortality of 41.7\% was observed (compared with 6.1\% in the whole cohort). This result needs to be treated cautiously given the absence of a replication cohort. Nevertheless. the underlying principle is an important one; subgroups of individuals with certain genotypes might benefit from therapies that have been evaluated as non-efficacious in a heterogeneous cohort. 

A further example comes in the form of a post-hoc analysis of the Vasopressin vs Norepinephrine as Initial Therapy in Septic Shock (VANISH) trial \parencite{Antcliffe2019}. In this study, patients were randomised to receive norepinephrine or vasopressin followed by hydrocortisone or placebo. Patients were also categorized into two groups based on their peripheral blood leukocyte gene expression pattern, SRS1 (sepsis response signature 1) and SRS2. The authors observed an interaction between assignment to hydrocortisone or placebo, and SRS endotype (p=0.02) whereby hydrocortisone administration was associated with increased mortality in those with an SRS2 phenotype (odds ratio = 7.9; 95\% CI 1.6-39.9). This result is biologically plausible as individuals with the SRS2 phenotype were relatively immunocompetent compared to the SRS1 phenotype; abolishing this immunocompetent phenotype through the administration of the corticosteroid is likely to have been harmful. Again, this result illustrates how a precision medicine approach may be beneficial in trials of sepsis therapy by using transcriptomic signatures to select individuals who might benefit from a particular treatment.

\section{Omics-based approaches and the importance of the pathogen}

Omics-based methods have applied high-throughput techniques to enable understanding of the molecular mechanisms of sepsis from gene level to biological phenotype. Genomics, transcriptomics, epigenomics, proteomics, metabolomics, and metagenomics have all contributed to our understanding of sepsis pathophysiology and disease heterogeneity. 

In this section, I will illustrate how omics-based approaches as applied to sepsis will remain limited while studies using such approaches focus on the host to the exclusion of the pathogen. Examples from sepsis and other infectious diseases will be used to illustrate where efforts to account for the degree of heterogeneity involving the pathogen have led to valuable insights into host biology. 

\subsection{Genomics}
Three decades ago, a seminal epidemiological study \parencite{Sorensen1988} documented the surprising observation that premature mortality from severe infection is strongly heritable. Since then, a number of genomic approaches have been applied to studying the association between host genetics and sepsis susceptibility or disease outcome. Many of the results have been conflicting or underwhelming; a systematic review \parencite{Clark2006} of 76 candidate gene studies in sepsis assessed the majority of studies to be of low to moderate quality. Using Bayesian statistics, the authors calculated that 30 of the 32 SNPs associated with sepsis had at least a 50\% probability of being a false-positive finding if the prior probability of a true association was set at a liberal cutoff of 0.01.

Currently, there are only two reported GWAS of sepsis in the literature \parencite{Rautanen2015} \parencite{Scherag2016}; each of the studies identified different associations with 28-day survival. In the first study \parencite{Rautanen2015}, the reported association involved a SNP (rs4957796) located in an intron of the Fps/Fes related tyrosine kinase \textit{(FER)} gene on chromosome 5. Interestingly, this association reached significance only when cause of sepsis and microbiology was accounted for. Statistical significance was reached only when patients with sepsis from pneumonia were analysed independently from those with intra-abdominal infection (p combined=5.6x10$^{-8}$), with effect size increasing when bacterial cases of pneumonia were analysed separately (odds ratio reduced from 0.56 to 0.40, indicating increased effect size). \textit{FER} is implicated in sepsis-relevant pathways involving neutrophil chemotaxis and endothelial permeability; electroporation-mediated gene therapy of a plasmid containing the protective SNP imrpoves survival in murine models of traumatic lung contusion and pneumonia \parencite{Dolgachev2016}. However, two independent studies (a second genome-wide association study \parencite{Scherag2016} and a targeted validation study \parencite{Schoneweck2015}) have failed to replicate this association involving the \textit{FER} gene. 

The second GWAS \parencite{Scherag2016} analysed a wide range of infection types (lung, abdominal, urogenital, bone, soft tissue, and wound infections) and identified a different association (rs117983287) localising to the vacuolar protein sorting 13 homolog A \textit{(VPS13A)} gene on chromosome 9 (p=8.16x10$^{-8}$). Failure of replication between these genome-wide association studies, despite similar methods and outcome of interest, is potentially attributable to the different causes of sepsis investigated in each study. 

\subsection{Transcriptomics}
Transcriptomic studies in sepsis have aimed to define disease subphenotypes with distinct host response features. However, challenges in dissecting the various sources of disease heterogeneity have limited efforts to apply these studies to sepsis biomarker development. 

Studies comparing host transcriptomic signatures between different microbiological classes have yielded varying results. One study comparing neutrophil gene expression in ICU patients with Gram-positive and Gram-negative sepsis failed to identify significant differences \parencite{Tang2008}. This result is unsurprising given the small patient numbers (18 Gram-positive patients, 25 Gram-negative patients) and the diverse range of sepsis causes studied. For example, the Gram-negative group included many infection types (respiratory, intra-abdominal, urinary, and central nervous system) from ten different bacterial species. A unique transcriptome profile may have been observed if the authors had refined their analysis to a particular infection (e.g. respiratory) or pathogen type in a larger cohort of patients. 

A similar approach applied genome-wide gene expression profiling to peripheral whole blood in sepsis caused by acute respiratory illness. Three gene sets were defined that classified patients into bacterial, viral, co-infected, and non-infected causes with a classification accuracy superior to that of procalcitonin (87\% vs 78\%; p$<$0.03) \parencite{Tsalik2016}. 

Cheng and colleagues \parencite{Cheng2016} studied critically ill patients with sepsis who had proven \textit{Escherichia coli} bacteraemia and candida fungaemia. Analysis of peripheral blood gene expression microarray data showed that the majority of genes (5977) with significant differential expression from healthy volunteers (adjusted p$<$0.05) were common to both pathologies. Genes encoding proteins within pathways relevant to glycolysis and oxidative metabolism were upregulated in both groups. However, a unique transcriptional response specific to each pathogen was also seen (\textit{E. coli} bacteraemia, 1718 genes; candida fungaemia 830 genes). 

Multi-cohort analysis of publicly available transcriptomic data from patients with acute infection provides further evidence for distinct gene expression patterns depending on the pathogen \parencite{Sweeney2016}. Analysis of datasets from eight cohorts of patients enabled derivation of a seven-gene set, which discriminated bacterial from viral infection in 30 independent cohorts. In combination with a separate 11-gene set that discriminates infection from no infection \parencite{Sweeney2015}, the authors derived an antibiotics decision model that had 94\% sensitivity and 59.8\% specificity for bacterial infection. 

All these studies highlight the importance of accounting for underlying microbiological heterogeneity in studies of host gene expression in sepsis, such that any differences between study groups can be clearly distinguished from the signature attributable to the pathogen. 

\subsection{Regulation of gene expression}
Regulation of gene expression is dynamic, involving highly coordinated processes from transcription to post-translational stages. An extensive review of published GWAS identified that the majority of trait-associated and disease-associated SNPs were located in non-protein coding regions of the genome, with 88\% situated in intergenic or intronic regions \parencite{Hindorff2009}. These genetic variants are believed to contribute to the heritability of inter-individual variation in gene expression. In fact, SNPs associated with complex traits are more likely to be expression quantitative loci (eQTL) than frequency-matched SNPs \parencite{Nicolae2010}. Given that eQTL show strong heritability \parencite{Wright2014}, these regulatory genetic variants could reflect the major selective pressure that pathogens have exerted on human genetics. 

Quantitative trait loci mapping approaches define the association of a SNP with an intermediate phenotype (e.g. levels of a transcript (eQTL), protein, or metabolite). These approaches have convincingly shown the importance of cellular context and environment on the activity of a regulatory variant. For example, in-vitro studies of human monocytes exposed to biological stimuli relevant to bacterial (lipopolysaccharide) and mycobacterial or viral (interferon-$\gamma$) infections have shown that exposure to these stimuli is necessary to show the association of SNPs with the expression of innate immune system-related genes \parencite{Fairfax2014}. In some examples, the direction of effect on gene expression (i.e. an increase or decrease in gene expression) can even differ between different conditions. A similar approach applied to human primary dendritic cells exposed to mycobacterium tuberculosis identified 198 loci associated with gene expression that were not observed in unstimulated cells, reflecting the effect of the host-pathogen interaction \parencite{Barreiro2012}. 

These observations are highly relevant to studies of sepsis patients, where we must consider the biological context set by a particular infecting pathogen and its influence on underlying host genomic modulation of gene expression. In a study of GAinS ICU patients with sepsis from CAP \parencite{Davenport2016} there was evidence of eQTL in genes associated with viral respiratory infection. This was observed even though only 9\% of patients within this cohort had CAP of confirmed viral cause; this analysis would have been enhanced by more detailed microbiological phenotyping of the cohort. 

The relevance of non-coding RNAs in the regulation of gene expression has become increasingly recognised. MicroRNAs (miRNAs) have been of particular interest as potential biomarkers in sepsis. However, conflicting results pose a challenge for the interpretation and application of these studies. For example, significantly reduced miR-223 levels were observed in one cohort of patients with sepsis, compared with patients with systemic inflammatory reponse syndrome and healthy individuals \parencite{Wang2010}, while elevated miR-223 levels were observed in another cohort of patients with sepsis compared with healthy individuals \parencite{Wang2012}. In a third study \parencite{Benz2015}, no differences in miR-223 levels were found between individuals with and without sepsis. The authors of the third study attribute these conflicting results to important difference in methods of normalisation. However, differences in cause of infection deserve consideration. The first of these three studies included only ICU surgical patients with intra-abdominal and trauma-related infection, whereas the other two recruited patients with respiratory infection and a broader range of infective pathologies. Even within the context of different strains of the same virus, significant differences in miR-223 expression can be observed. In mice infected with a reconstructed pandemic 1918 H1N1 influenza A virus, distinct patterns of lung tissue miRNA expression were observed compared with those infected with a seasonal H1N1 influenza A virus (strain A/Texas/36/91) \parencite{Li2010}. Specifically, levels of miR-223 were nearly three-times higher with the pandemic strain than with the seasonal strain.

\subsection{Proteomics and metabolomics}
Advances in mass-spectrometry-based methods enables large-scale analysis of the protein and metabolite composition of biological samples in sepsis research. There is the potential for unique functional insights beyond those afforded by genomics and transcriptomics since there is no direct association between mRNA expression and protein or metabolite levels.

Langley and colleagues \parencite{Langley2013} studied patients presenting to the emergency department with suspected community-acquired sepsis. Unique plasma proteomic and metabolomic profiles were seen in the three groups of patients: sepsis survivors, sepsis non-survivors, and individuals with a non-infective systemic inflammatory response syndrome. Non-survivors had profiles indicative of impaired mitochondrial fatty acid $\beta$ oxdiation. The authors of this study developed a prognostic logistic regression model based on seven parameters (four carnitine esters, lactate, age, and haematocrit). This model was validated in two separate cohorts and successfully classified sepsis survivors and non-survivors with an accuracy of 85\%, superior to that of lactate, Sequential Organ Failure Assessment, or Acute Physiology and Chronic Health Evaluation II scores. Interestingly, no substantial differences were seen between the plasma metabolome and proteome of patients with sepsis due to \textit{Streptococcus pneumoniae}, \textit{Staphylococcus aureus} and \textit{Escherichia coli}. The authors postulated that this might have been because of heterogeneity with respect to infection site, and the possibility that subtle differences were overwhelmed by a generalised septic response. Indeed, in a different cohort of patients with Gram-positive and Gram-negative sepsis, ELISA-based quantification of a more limited panel of 11 plasma cytokines showed higher interleukin-1$\beta$, interleukin-6 and interleukin-18 concentrations in the Gram-positive group compared with the Gram-negative group \parencite{Feezor2003}. In another study \parencite{Huang2014}, recruiting a more homegeneous cohort of Gambian children with pneumonia, a mass-spectrometry-based proteomic analysis identified 42 proteins that differentiated severe from non-severe pneumonia and non-severe pneumonia from controls \parencite{Huang2014}. One of these proteins was neutrophil gelatinase-associated lipocalin (lipocalin-2), which discriminates pneumonia of probable bacterial cause from viral cause. Children with plasma concentrations of lipocalin-2 more than 163 ng/ml were nine times more likely to have a positive blood culture with a clinically significant isolate. Proteomic and metabolomic studies of sepsis represent a rich, untapped source of potential disease biomarkers for the clinical setting. 

\subsection{Epigenetics}
From the perspective of evolutionary biology, the co-existence of pathogen and host is a major selective pressure on the genetic diversity of both organisms \parencite{Sironi2015}. While random point mutations in DNA (SNPs) are a key source of diversity, the role of epigenetic changes in contributing to more rapid changes to organism phenotype is being increasingly recognised \parencite{Rando2007}. Both viruses \parencite{Paschos2010} and bacteria \parencite{Hamon2008} have the potential to induce epigenetic changes in humans, modulating the biological interaction between pathogen and host with significant consequences to disease course.

For example, the H3N2 influenza A virus carries a histone-like sequence in its non-structural protein 1 (NS1) tail. This allows it to interact with the host epigenome;   binding of the NS1 protein to the human polymerase associated factor 1 transcription elongation complex allows the virus to target sites of actively transcribed antiviral genes, suppressing the antiviral response \parencite{Marazzi2012}. Interestingly, the H1N1 influenza A virus does not possess this histone-like NS1 tail, which might explain the different disease phenotypes seen between various strains.

There are few epigenetics studies of sepsis. However, there is substantial corroborative evidence to support epigenetics as a promising approach to advance our understanding of the disease. In an animal model of acute lung injury and sepsis, anaesthetised mice were exposed to a dual pulmonary insult of aspiration of a \textit{Staphylococcus aureus} culture and mechanical ventilation \parencite{Bomsztyk2015}. After only 6 hours, there was decreased expression of the angiogenic genes \textit{ANGPT1, TEK,} and \textit{KDR} in the lung, kidney, and liver. Chromatin immunoprecipitation assays showed a decrease in relative RNA polymerase II abundance and histone deacetylation at these genes, providing evidence for the role of epigenetic changes in contributing to sepsis-induced endothelial dysfunction.

In the previously mentioned study of gene expression in adult ICU patients with sepsis due to CAP \parencite{Davenport2016}, the locations of the observed sepsis eQTL showed substantial overlap with epigenetic marks observed in monocytes stimulated with bacterial lipopolysaccharide. This overlap included deoxyribonuclease (DNase) I hypersensitive sites (i.e regions of chromatin where DNase I activity results in the DNA being accessible to transcription factor binding) and histone marks associated with enhancer and promoter regions (i.e. specific covalent modifications such as H3K27ac, H3K4me1, H3K4me3).

In summary, results from studies in animal models and in humans with sepsis suggest a strong role for epigenetic changes in sepsis pathophysiology. However, to translate these changes to patient benefit (e.g. identification of novel therapeutic targets), these epigenetic mechanisms will need to be dissected further. Studies that take account of specific infection types will be an important way to achieve this aim.

\subsection{Metagenomics}
Unique host and pathogen factors interact to result in the heterogeneous sepsis immune response. However, this host-pathogen interaction does not occur in isolation but within a rich microbial context. The term pathobiome illustrates this concept - that a microorganism's pathogenicity depends not just on its specific virulence factors, but also on host factors, environmental factors, and the microbial community of which it is a part \parencite{Vayssier-Taussat2014}. 

Advances in metagenomics has been facilitated by the increasing affordability of high-throughput sequencing. Of relevance to sepsis is the mechanisms by which one microorganism (or a community of microorganisms) can influence the interaction of another microorganism with the host. This synergism between multiple pathogens is well-recognised in influenza virus infection, where adherence of \textit{Streptococcus pneumoniae} to the respiratory epithelium is facilitated by viral neuraminidase \parencite{Peltola2005}. Metagenomics now enables the consideration of synergism beyond a handful of organisms, by revealing new mechanisms through which the whole microbiome influences host susceptibility to infection.

Mucosal surfaces represent a critical interface for the microbiome-host interaction; microorganisms previously considered to be pure commensals act both directly and indirectly at sites such as the intestinal and respiratory epithelium to protect the host from potentially pathogenic organisms. Directly, the intact microbiome provides a barrier effect by limiting the supply of essential resources at these sites. Indirectly, the microbiome interacts with immune cells at mucosal surfaces, leading to the modulation of key innate and adaptive immune pathways \parencite{Thaiss2016}. In murine models, the bacterial component of the gut microbiome plays an essential part in enabling persistent norovirus infection by limiting the efficacy of interferon-$\gamma$ mediated innate immunity \parencite{Baldridge2015}. Adaptive immunity is also mediated by the microbiome, with the response to influenza virus shown to depend on a healthy lung bacterial microbiome. Mice receiving a 4-week course of oral antibiotics developed marked dysbiosis of the lung microbiome, leading to defective CD4 T-cell, CD8 T-cell, and B-cell mediated immunity to subsequent intranasal influenza virus challenge \parencite{Ichinohe2011}. This suggests that antibiotics might be not just unnecessary, but actually harmful in cases of viral respiratory infection with potentially serious implications in patients with chronic lung conditions (e.g. chronic obstructive pulmonary disease or cystic fibrosis), who frequently receive antibiotics as prophylaxis or for treatment of exacerbations. 

Large-scale endeavours such as the National Institutes of Health Human Microbiome Project \parencite{Peterson2009} have revealed the extent to which the microbiome differs between healthy individuals. This diversity is often lost in critical illnesses such as sepsis, in which both disrupted host pathophysiology and clinical interventions to manage the underlying condition result in striking changes to the microbiome \parencite{Dickson2016}. There is convincing evidence to suggest that host genetic variants interact with the microbiome to result in the development of immune-related disease; for example, K/BxN transgenic mice do not develop inflammatory arthritis in a germ-free environment since segmented filamentous bacteria in the gut are essential to T-helper-17 cell activation and subsequent joint inflammation \parencite{Wu2010}. This could have implications in sepsis, in which the relevance of host genetics to disease might be better understood in the context of inter-individual differences in the microbiome.

There is potential for metagenomic analysis of the microbiome to yield prognostic information in sepsis: in conditions as diverse as bronchiectasis \parencite{Rogers2014} and metabolic syndrome \parencite{LeChatelier2013}, the microbiome has been shown to correlate with disease phenotype and outcome. Metagenome-wide association studies \parencite{Zhang2015} that identify associations between microbial genes and disease traits have found that restoration of dysbiosis in the dental microbiome is associated with good response to disease-modifying anti-rheumatic drugs in rheumatoid arthritis. It is not inconceivable that microbiome-based tests could similarly be identified as biomarkers for early diagnosis, patient stratification, and evaluation of treatment response in sepsis. Additionally, the microbiome represents an attractive therapeutic target in sepsis, with murine models showing a reduction in criculating aged neutrophils following antibiotic depletion of the gut-microbiota, with corresponding improvements in survival from endotoxin-induced septic shock \parencite{Zhang2015a}. As an important source of heterogeneity in sepsis, characterisation of the microbiome and integration of this second genome \parencite{Grice2012} into other omics-based approaches should be a key priority in further sepsis research.

\section{New opportunities in clinical microbiology}
A USA-based epidemiological study \parencite{Gupta2016} of nearly 7 million sepsis patients hospitalised between 2001 and 2010 estimated the incidence of culture-negative sepsis at 47\%, with culture negativity identified as an independent predictor of mortality. Making a microbiological diagnosis is a key clinical priority, enabling effective antimicrobial therapy within the framework of responsible stewardship. Emerging new technologies bring the potential of more rapid and detailed resolution of microbiology in sepsis, and can be classified into one of three main approaches: PCR-based techniques, mass-spectrometry-based methods, and nucleic acid sequencing (next-generation or high-throughput sequencing) methods.

\subsection{Multiplex PCR}
Multiplex PCR platforms enable the simultaneous detection of multiple pathogens with high sensitivity and specificity. Antimicrobial susceptibility data can be provided and there is potential for automation. Point-of-care platforms are being increasingly commonplace in the clinical setting with the advantage of rapid turnaround times and user-friendly sample processing. 

The BioFire FilmArray (bioMerieux, Marcy l'Etoile, France) respiratory \parencite{Xu2013} and meningitis/encephalitis \parencite{Lee2019} panels are two examples of disease specific multiplex PCR platforms. The respiratory panel \parencite{Xu2013} tests for 15 viral pathogens in respiratory specimens and was implemented in a regional paediatric hospital in the USA with a median turnaround time of 1.4 hours. Over 2,500 specimens were tested by FilmArray and a direct fluorescence assay in parallel. The FilmArray panel detected rhinovirus in 20\% of samples and coronavirus in 6\% of samples, both organisms were not part of the direct fluorescence assay panel. Whilst the respiratory panel only tests for viruses, the meningitis/encephalitis panel tests for 14 pathogens, including 6 bacteria, 7 viruses, and 1 fungal species from cerebrospinal fluid \parencite{Lee2019}. Forty-two individuals presenting to the emergency department with relevant symptoms were tested using the FilmArray platform; six positive samples were detected with an 88\% agreement rate with conventional microbiology testing.

There are two main disadvantages with these platforms. Firstly, detection is limited to pathogens in the pre-specified probe panel, which are limited in their scope. Therefore, the platforms are limited in geographical locations where there is a high prevalence of infection caused by microorganisms not included in the panel. Secondly, false-negative results may arise where variation among pathogen genomes leads to failure to detect the organism. For example, a variant strain of \textit{Chlamydia trachomatis} in Sweden led to false negative testing by PCR \parencite{Ripa2006} because the variant strain had a deletion of 377bp in the cryptic plasmid, the region targeted by PCR testing.

\subsection{Mass spectrometry}
There are generally two methods of mass spectrometry applied to microbiological testing: matrix-assisted laser desorption ionisation-time of flight mass spectrometry (MALDI-TOF) and electrospray ionisation-mass spectrometry (ESI-MS) \parencite{Buchan2014}. The difference in the two techniques lies in the generation of the ions. In MALDI-TOF, the analyte is allowed to dry before being overlaid by a weak acid matrix material. It is then exposed to a laser which ionises and desorbs it from the sample plate. The created ions are then accelerated through a vacuum by the application of an electrostatic field. In contrast, analytes need to be in the liquid phase for ESI-MS. The solute is passed through a heated capillary and exposed to a high voltage to generate an aerosol of ions. 

Huang and colleagues compared MALDI-TOF and antimicrobial stewardship team (AST) intervention with routine clinical care in individuals with bloodstream infection \parencite{Huang2013}. Routine clinical care included analysis of blood culture results using the VITEK-2 system (biochemical testing for antimicrobial susceptibility information) without AST intervention. In the study group, the AST intervention consisted of provision of evidence-based antimicrobial recommendations after receiving a positive result. The intervention group showed significantly decreased time to organism identification, decreased time to effective antimicrobial therapy, decreased 30-day mortality and decreased ICU length of stay, compared with the pre-intervention control group.

ESI-MS has also been studied in individuals with bloodstream infection \parencite{Vincent2015}. In 616 individuals with bloodstream infection, ESI-MS identified a pathogen in 37\% of individuals compared with a 11\% diagnostic rate for blood cultures. An expert panel was enlisted to provide independent clinical analysis and concluded that ESI-MS could potentially have altered treatment in up to 57\% of patients.

\subsection{Nucleic acid sequencing}
See next section (Section \ref{sec:clinmetagenomics}) for a description of short-read nucleic acid sequencing.

\subsection{Nanopore sequencing}
Nanopore sequencing is an emerging NGS technology that performs long-read sequencing with real-time sequence analysis \parencite{Jain2016}. Commercially available platforms include the MinION (1 flow cell), GridION (5 flow cell capacity), and PromethION (48 flow cell capacity). The MinION has been successfully applied to outbreak surveillance, e.g. in epidemiological outbreak surveillance of Ebola virus in west Africa \parencite{Quick2016} and Salmonella in a UK-based inpatient setting \parencite{Quick2015}.

Examples of application to diagnostics in clinical syndromes remains limited due to inherent challenges; MinION sequencing has been applied to urinary tract infections \parencite{Schmidt2017}, prosthetic joint infections \parencite{Sanderson2018}, and lower respiratory infections \parencite{Charalampous2019}. Schmidt and colleagues \parencite{Schmidt2017} analysed 10 heavily infected urine specimens ($>$10$^7$ cfu/ml) and 5 specimens of healthy urine spiked with multi-drug resistant \textit{Escherichia coli}. With a turnaround time of 4 hours, there was 100\% agreement between MinION sequencing result, Illlumina sequencing, and clinical microbiology. In addition, 51/55 resistance genes detected by Illumina sequencing were found by minION sequencing. However, the authors noted a number of limitations with their study, including the fact that only heavily infected urine was analysed, the cost of sequencing (only one sample was sequenced per flow cell) and the poor identification of allelic variants and resistance-conferring mutations due to poor base-calling accuracy. Compared to Illumina sequencing, the potential for scalability in terms of number of samples and data volumes remains relatively limited in Nanopore sequencing.  

\section{Clinical metagenomics} \label{sec:clinmetagenomics}
Clinical metagenomic next-generation sequencing has been described as the characterisation of \textit{``all DNA and/or RNA present in a sample, enabling analysis of the entire microbiome as well as the human host genome or transcriptome in patient samples"} \parencite{Chiu2019}. 

\subsection{Applications}
\textbf{Infectious disease diagnostics.} NGS approaches can be untargeted or targeted in their approach. Untargeted approaches involve the sequencing of the entire DNA and/or RNA component of a sample. Typically, $>$99\% of the reads obtained are human in origin, so sensitivity for a pathogen can be a critical issue. However, advantages include the unbiased nature of the approach, enabling comprehensive analysis of all pathogens in a single assay. 

Targeted approaches include the following three approaches: (i) targeted PCR amplification of a conserved region (e.g. 16S ribosomal RNA (rRNA) gene amplification for bacteria \parencite{Watanabe2018}, and 18S rRNA and internal transcribed spacer gene amplification for fungi \parencite{Wagner2018}), (ii) targeted PCR amplification of a whole genome using tiled primers \parencite{Quick2017}, and (iii) hybrid-capture based techniques whereby metagenomic libraries are subjected to hybridisation using capture "bait" probes \parencite{Bonsall2015}. Targeted approaches are able to increase the proportion of pathogen reads in the sequence data, thereby increasing sensitivity with potential reductions in cost of sequencing because less total sequencing reads are required to achieve a given depth of pathogen coverage.

\textbf{Clinical microbiome analysis.} There is increasing awareness of the role of the microbiome in the pathogenesis of both acute and chronic illnesses. One group charcterised the microbiome of blood in sepsis (n=62) patients and healthy volunteers (n=23) \parencite{Gosiewski2017}. The authors observed striking differences between the taxonomic composition of the two groups; the microbiome of healthy volunteers was composed of mainly anaerobic bacteria whereas the microbiome of septic patients was composed of mainly aerobic and microaerophilic microorganisms. There was decreased representation of the Actinobacteria phyla in sepsis patients and decreased representation of the Proteobacteria phyla in healthy volunteers. 
 
\textbf{Human transcriptomic analysis.} Although clinical metagenomics focuses on the microbial reads recovered from a sample, analysis of human gene expression is also possible if RNA libraries have been constructed from the clinical samples. This incidental RNA-seq data has the potential to inform analysis of pathogen reads. For example, infective and non-infective pathologies can be differentiated by combined host and pathogen read data analysis. Clinical metagenomic NGS of tracheal aspirate samples enabled differentiation of individuals with acute respiratory failure from lower respiratory tract infections versus those with non-infectious causes based on pathogen, microbiome diversity, and host gene expression metrics \parencite{Langelier2018a}. Also, dual transcriptome analysis enables improved understand of host-pathogen interactions. Analysis of Gambian children infected with \textit{Plasmodium falciparum} revealed distinct human and parasite gene expression profiles associated with severe malaria phenotypes \parencite{Lee2018}. Up to 99\% of human differential gene expression was driven by differences in parasite load and coexpression analyses revealed interactions between host and parasite, with marked co-regulation of translation genes in severe malaria. Furthermore, RNA-seq can be especially useful in cases of infection where the causative pathogen is only transiently present in the host, e.g. Lyme disease \parencite{Marques2015} or Zika virus infection \parencite{Landry2017}, since distinct pathogen-specific human transcriptomic signatures may be identified.

\textbf{Applications in oncology.} Sequencing of tumour samples can be used to identify viruses associated with cancer (e.g. herpesviruses, papillomaviruses, and polyomaviruses) and also to identify virus-host interactions. For example, metagenomic NGS enabled the identification of a previously unknown polyomavirus, Merkel cell polyomavirus, in skin tissue analysis of Merkel cell carcinoma (a rare but aggressive dermatological cancer) \parencite{Feng2008}. In addition, targeted metagenomic sequencing enabled eight whole genomes of Epstein-Barr virus to be sequenced from primary nasopharyngeal carcinoma biopsy specimens \parencite{Kwok2014}. Clinical metagenomics may also inform the treatment of cancers associated with viral infection. For example, Kanwal and colleagues \parencite{Kanwal2017} showed that in hepatitis C virus patients treated with direct-acting antiviral agents, those with sustained virological response showed a reduction in the risk of hepatocellular carcinoma.

\subsection{Hybrid-capture based techniques}
Of the three approaches to targeted metagenomics described above, the hybrid-capture based technique is the most versatile as it does not depend on the specificity of highly conserved primers for PCR amplification. The capture procedure is typically applied after nucleic acid extraction and library preparation. Here, RNA baits are preferable to DNA baits as RNA:DNA duplexes hybridise with greater efficiency and stability than DNA:DNA hybrids \parencite{Lesnik1995}. The hybridisation can be carried out on a solid support (i.e. array-based) \parencite{Schuenemann2018}, or more commonly, in-solution \parencite{Briese2015}. For the in-solution technique , biotinylated baits bind the target of interest \parencite{Gnirke2009}. Then, Streptavidin coated magnetic beads are added to the solution, which bind to the biotinylated baits. Subsequent washing steps lead to non-specific unbound molecules being washed away before the sample is sent for NGS. 

Hybrid-capture based techniques have been applied to clinical metagenomic NGS of bacteria \parencite{Allicock2018}, viruses \parencite{Depledge2011} \parencite{Briese2015} \parencite{Wylie2015}, fungi \parencite{Amorim-Vaz2015}, and parasites \parencite{Bright2012}. 

For bacterial sequencing, the BacCapSeq platform includes a probe set comprised of 4.2 million oligonucleotide probes which enable detection and characterisation of bacteria, virulence determinants and antimicrobial resistance genes \parencite{Allicock2018}. The capture resulted in up to a 1000-fold increase in sensitivity with blood samples.

For viral sequencing, the first example of hybrid-capture based NGS involved a study which targeted three full length herpesvirus genomes (Varicella-Zoster Virus, Epstein-Barr virus and Kaposi's sarcoma-associated Herpesvirus) \parencite{Depledge2011}. A range of clinical sample types were tested across 13 samples and full length herpesvirus genomes reconstructed at high read depth. Other examples of hybrid-capture based NGS applied to viruses include ViroCap \parencite{Wylie2015} and VirCapSeq-VERT \parencite{Briese2015}. Virocap includes a panel of probes designed to enrich for nucleic acid from 34 families of DNA and RNA viruses (190 viral genera and 337 species) that infect vertebrate hosts, excluding human endogenous retroviruses \parencite{Wylie2015}. The authors showed that the probes only required a minimum of 58\% probe-target homology for enrichment and sequencing. VirCapSeq-VERT includes 2 million probes covering 207 viral taxa that infect vertebrates \parencite{Briese2015}. The authors demonstrated that the capture resulted in a 100- to 10,000-fold increase in viral reads from blood and tissue homogenates compared to conventional Illumina sequencing using established virus enrichment procedures (filtration, nuclease treatments, and RiboZero rRNA subtraction). Novel viruses were also detected where their genomes were approximately 60\% similar to probes included in the capture library.

For fungal sequencing, Amorim-Vaz and colleagues \parencite{Amorim-Vaz2015} designed a set of 55,342 probes covering 6,094 \textit{Candida albicans} open reading frames. Results showed approximately 1000-fold enrichment of \textit{C. albicans} reads in biological samples and a detection of more than 86\% of its genes. 

For parasite sequencing, \textit{Plasmodium vivax} enrichment enabled an increase in sequencing yield from 0.5\% to a median of 55\%, with 5X coverage across 93\% of the \textit{P. vivax} genome \parencite{Bright2012}.

\subsection{Application to sepsis}
Apart from the work described in this thesis \parencite{Goh2019}, there are several recent examples in the literature of clinical metagenomics as applied to adult sepsis patients. None of these examples include the simultaneous sequencing and analysis of bacterial, DNA viral, and RNA viral reads. 

Blauwkamp and colleagues \parencite{Blauwkamp2019} performed microbial cell-free DNA sequencing from the plasma of 348 adult patients presenting to the Emergency Department who met sepsis alert criteria. Next-generation sequencing (NGS) yielded a positive diagnosis in 49\% of individuals, compared to an 18\% diagnostic yield from blood cultures and 38\% diagnostic yield from all conventional microbiological testing combined. The method only enabled identification of bacteria, DNA viruses, fungi, and eukaryotic parasites. NGS was particularly helpful in identifying a cause of sepsis in patients who had received preceding antibiotics when compared to conventional microbiological testing. The authors also describe their experience of NGS processing of the first 2000 plasma samples submitted to their Clinical Laboratory Improvement Amendments certified and College of American Pathologists accredited laboratory. They achieved a 98\% rate of sample reporting within one day after sample receipt.

In a smaller study, Grumaz and colleagues \parencite{Grumaz2019} also applied cell-free DNA sequencing to plasma from septic shock patients (n=48). NGS yielded a diagnosis in 72\% of individuals compared to a 33\% diagnostic rate from blood cultures. A clinical expert panel reviewed the NGS results; 96\% of NGS results were deemed plausible and were judged to have led to a change to more adequate therapy in 53\% of cases.

In six Vietnamese hospitals, NGS was applied to 492 clinical samples from 386 patients with community acquired sepsis \parencite{Anh2019}. A range of sample types were sequenced including serum, nasal/throat swabs, stool and cerebrospinal fluid. Only sequencing reads aligning to viral sequences were analysed. The authors confirmed each positive NGS result with PCR and considered only those validated by PCR as positive. 21 viral species known to be infectious in humans were identified in 13.4\% of patients.

Finally, a study of Chinese ICU patients with sepsis (n=78) described the preparation of DNA libraries from plasma and Ion Torrent sequencing \parencite{Long2016}. NGS yielded a 31\% diagnostic rate versus 13\% from blood cultures. 

\subsection{Challenges}
There are a number of challenges associated with clinical metagenomics, which will be described in this section.

\textbf{Sensitivity.} As discussed above, typically $>$99\% of metagenomic NGS reads generated originate from the host. This is because the majority of nucleic acid in clinical specimens is human in origin, and also because human genomes are far larger than that of bacteria and viruses. Strategies to increase sensitivity for microbial reads include targeted metagenomic approaches (described above) as well as other purification or enrichment procedures to increase pathogen yield. For bacterial pathogens, the most common first step is culture, which can favour contaminating, culturable or fast-growing organisms. For viral pathogens and some bacteria, yield-maximising methods include filtration to remove host cells \parencite{Allander2001}, sample treatment with nucleases to digest nucleic acid not protected within cells or virions \parencite{Allander2001} \parencite{Charalampous2019}, and high-speed gradient centrifugation to concentrate virus particles \parencite{Breitbart2005}. Each of these procedures reduces throughput and may bring about bias.

\textbf{Interpretation of positive results.} In the context of a metagenomic positive result, it can be challenging to differentiate infection from colonisation or contamination. In certain samples (e.g. respiratory specimens), the normal microbiota can be rich and complicate interpretation of results. Systematic PCR testing of nasopharyngeal or bronchoalveolar lavage specimens from patients in intensive care, admitted with pneumonia, identified respiratory viruses in over a third of patients, the relevance of which remains uncertain \parencite{Choi2012}. One potential solution is the use of spike-in based calibration to enable quantitative analysis through conversion of percentage reads to colony-forming units per milliliter \parencite{Stammler2016}. Another approach is to assess the host immune response by simultaneous analysis of the human transcriptome. Langelier and colleagues evaluated a composite metric of immunity genes in haematopoietic cellular transplant patients as a biomarker of active infection, using this to differentiate colonisation from active infection \parencite{Langelier2018}. 

The use of controls (healthy individuals as well as no-template controls) can also be a useful strategy, especially for differentiating causative pathogens from extraneous sources of DNA ubiquitous in commonly used reagents used in the extraction and library preparation procedures \parencite{Salter2014}. For example, Grumaz and colleagues \parencite{Grumaz2016} assigned a sepsis indicating quantifier score to each identified microbe in a sample, to indicate the probability of it being a true positive finding (on the basis of the number of reads mapping to the microbe's reference genome in the patient sample, relative to healthy individuals). A further approach is to evaluate the proportion of the genome covered by the reads mapping to an organism; where reads are localised to limited regions of the genome, this is more likely to represent contamination when compared to reads spanning larger regions of the genome.

\textbf{Turnaround times.} Time from sample receipt to data generation can vary, and has been reported to be anywhere from 6 hours to 7 days (average 48 hours) \parencite{Simner2018}. These long turnaround times arise because of laborious library preparation stages essential to making a clinical sample suitable for sequencing as well as time taken for the generation of sequence data. However, automated processes are becoming increasingly commonplace and new technologies, such as Oxford Nanopore Technologies, require less pre-processing and provide real-time data analysis \parencite{Quick2016}. 

\textbf{Data analysis.} A sizeable gap exists between the generation of NGS data and the analysis and processing of this data to generate clinically relevant information. Although bioinformatic pipelines have been implemented in clinical settings (e.g. the sequence-based ultrarapid pathogen identification SURPI pipeline \parencite{Wilson2019}), there is still lack of standardisation of data processing and analysis methods. In addition, bioinformatics packages and pipelines currently require a substantial degree of bioinformatics expertise that is not routinely available in clinical settings. 

The absence of well-curated databases also poses a challenge. Draft and partial genomes on NCBI (National Centre for Biotechnology Information) may lead to erroneous results. For example, false positive results may arise from mapping to low complexity regions in the reference sequence or to contaminants from database entries that contain reads to human DNA or sequencing adapters. Also, false negative results may arise due to incomplete or missing taxonomic representation in databases.

\section{Specific aims and objectives}
The aims of this thesis are to:

\begin{itemize}[leftmargin=*]

\item	\textbf{Develop a library preparation method for metagenomic sequencing of plasma samples 
(\hyperref[ch:Results1]{Ch. 3})} \\
Commercially available library preparation methods are available for either RNA or DNA, but not both within a single streamlined method. Sensitivity is likely to be a limiting factor in metagenomic sequencing from plasma; hybrid-capture based techniques provide an opportunity to improve sensitivity. I aim to:

		\begin{enumerate}
		\item Use probe-based enrichment to increase sensitivity for sequencing organisms relevant to sepsis due to CAP from plasma
		\item Optimise a library preparation method suitable for sequencing both DNA- and RNA-based organisms from plasma
		\item Evaluate performance of the library preparation method against a known positive control reference set
		\end{enumerate}
		
\item	\textbf{Improve definition of microbiological aetiology using targeted metagenomics, PCR-based pathogen analysis, and clinical data (\hyperref[ch:Results2]{Ch. 4})} \\
Various methods can be used to improve the microbiological diagnosis rate in GAinS CAP sepsis patients. I aim to improve microbiological classification of GAinS CAP sepsis patients through:

		\begin{enumerate}
			\item Application of targeted metagenomics to plasma samples
			\item Use of droplet digital PCR to assay \textit{Streptococcus pneumoniae} and Epstein-Barr virus from plasma samples
			\item Use of the Axiom Microbiome Array
		\end{enumerate}

\item	\textbf{Improve characterisation of disease heterogeneity in patients with no previous microbiological diagnosis through integrated application of metagenomic, transcriptomic and genomic techniques (\hyperref[ch:Results3]{Ch. 5})} \\
Integrated -omic approaches have the potential to enhance our understanding of the heterogeneous host response in sepsis. In this chapter, I aim to:
		\begin{enumerate}
			\item Characterise the extent and implications of EBV reactivation and integrate this with host transcriptomic data
			\item Investigate the association between \textit{Streptococcus pneumoniae} bacterial load and sepsis endotype (SRS status)
			\item Describe host transcriptomic signatures of viral infection, influenza infection, and \textit{Streptococcus pneumoniae} infection
			\item Investigate the association between host genotype (HLA type) and susceptibility to different classes of infection
		\end{enumerate}

		
\end{itemize}







